Criterion/Measurement.hs

```diff
@@ -1,9 +1,4 @@
-{-# LANGUAGE DeriveDataTypeable #-}
-{-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE Trustworthy #-}
-{-# LANGUAGE RecordWildCards #-}
-{-# LANGUAGE BangPatterns, CPP, ForeignFunctionInterface,
-    ScopedTypeVariables #-}
+{-# LANGUAGE PackageImports #-}
```

Comment:
Currently, there are two `Criterion.Measurement` files—one in `criterion`, and one in `criterion-measurement`—but they both export the exact same API. Moreover, the existence of both requires the use of `PackageImports` invasively, which I'm not fond of. I'd opt to just get rid of the `Criterion.Measurement` from `criterion` entirely. (It's a breaking change, after all.)